apps/framework-cli/src/framework/python/wrappers/consumption_runner.py

```diff
@@ -158,6 +158,7 @@ def execute(self, name: str, input_data: Any) -> Dict[str, Any]:
             }
     
     async def _start_workflow_async(self, name: str, input_data: Any):
+        # TODO: add retry & timeout
```

Comment:
We'd need to parse the configs again in python consumption, or figure out how to pass the configs to consumption api, or just call `moose-cli workflow run`. I wanted to get opinions before I go into a rabbit hole.

Reply:
Hum and there is the how to hot reload as well. 

I am thinking that points to needing to passing it in to be able to use the watcher in dev. Otherwise, we would have to do some watching logic here as well

Reply:
I was leaning to that option as well, but the tricky thing is we don't know which workflow they'll call from which api. So we'd have to pass all the configs, and might run into string length limit when passing the configs to api. The workaround was to serialize the configs to a .moose file. Lmk what you think.

Reply:
When I said passing it in I was thinking as a CLI argument when we start the consumption apis, since we trigger the restart of the consumption API server for all changes, we could hook changes to the script files to do the same think and pass in the config that way. It should not limit the length of what we pass in.

Then as I am writing this I am wondering if the context is the right one.